velodyne_pointcloud/include/velodyne_pointcloud/calibration.h

```diff
@@ -64,9 +65,10 @@ namespace velodyne_pointcloud {
   public:
 
     Calibration(bool info=true):
-      initialized(false), ros_info(info) {}
+      distance_resolution_m(0.002f), initialized(false), ros_info(info) {}
     Calibration(const std::string& calibration_file,
                 bool info=true):
+      distance_resolution_m(0.002f),
```

Comment:
Is this supposed to be overridable using rosparams at runtime? Where is the logic for that?

Reply:
It's a value that's included in the calibration YAML files. See the changes to velodyne_pointcloud/scripts/gen_calibration.py for the additional parsing logic.